lib/sgtn-client/api/base.rb

```diff
@@ -0,0 +1,92 @@
+# 
+#  Copyright 2019-2022 VMware, Inc.
```

Comment:
Why 2019? What's the relation to 2019?

Reply:
ahh, i use the recommended comments, i think it represents the whole project, not just this file